src/move.cpp

```diff
@@ -698,6 +698,11 @@ void ParallelTempering::_to_json(json& j) const {
         auto id = fmt::format("{} <-> {}", pair.first, pair.second);
         exchange_json[id] = {{"attempts", acceptance.size()}, {"acceptance", acceptance.avg()}};
     }
+    auto& exchange_json2 = j["exchangesmpi"] = json::object();
+    for (const auto& [mpI, exchange] : exchange_map) {
```

Comment:
I guess it's `mpI` due to clash with namespace? Perhaps rename to `_mpi` or alternatively something more verbose.